sporkfish/searcher/move_ordering.py

```diff
@@ -69,3 +70,38 @@ def evaluate(self, board: Board, move: chess.Move) -> float:
             ]
         else:
             return 0
+
+
+class KillerMoveHeuristic(MoveOrder):
+    def __init__(self, max_depth: int) -> None:
+        # Initialize a list to store up to two killer moves for each depth
+        # Using a fixed size per depth for simplicity; could be dynamic based on actual usage
+        self.killer_moves: List[List[Optional[chess.Move]]] = [
+            [None, None] for _ in range(max_depth)
+        ]
+
+    def add_killer_move(self, move: chess.Move, depth: int) -> None:
+        """
+        Add a move to the killer moves list for a given depth, ensuring it's not already present.
+        If the move is new, it replaces the oldest killer move.
+
+        Parameters:
+            move (chess.Move): The move to be added as a killer move.
+            depth (int): The depth at which the move proved to be a killer move.
```

Comment:
The doc strong is correct but we need *Sphinx* docs. Could you change these please?